cylc/flow/network/schema.py

```diff
@@ -769,7 +769,10 @@ class Meta:
     err_script = String()
     exit_script = String()
     execution_time_limit = Float()
-    host = String()
+    host = String(
+        deprecation_reason='Use the platform field'
+    )
```

Comment:
Turns out we can deprecate GraphQL fields, I think I'll just remove this one (since it will always return `None` and we are the only users ATM), however, worth knowing.

Reply:
Removed (not needed).